project3/index.html

```diff
@@ -0,0 +1,387 @@
+                    <input id="real-name" type="real-name" placeholder="" required>
+                </div>
+
+                <div class="alias">
+                    <label class="input-box" for="alias">Alias</label>
+                    <input id="alias" type="alias" placeholder="" required>
+                </div>
+
+                <div class="email">
+                    <label class="input-box" for="email" Email>Email</label>
+                    <input class="email" type="email" type="email" placeholder="" required>
+
+                </div>
+
+                <h2 class="personal">PERSONAL CHARACTERISTICS</h2>
+
+
+                <label class="age" for="age">Age<br></label>
+                <select class="agebox" name="" id="age">
+                    <option value="testing"></option>
+                    <option value="another"></option>
+                </select>
+
+                <fieldset class="field-margin">
+                    <legend class="category">Category</legend>
+
+                    <p class="margins">Please select as many as apply to your origin and how you gained your powers.</p>
+
+                    <input name="category" id="alien" type="checkbox">
+                    <label for="alien">Alien</label><br>
+
+                    <input name="category" id="cyborg" type="checkbox">
+                    <label for="cyborg">Cyborg</label><br>
+
+                    <input name="category" id="experiment" type="checkbox">
+                    <label for="experiment">Experiment</label><br>
+
+                    <input name="category" id="human" type="checkbox">
+                    <label for="human">Human</label><br>
+
+                    <input name="category" id="inhuman" type="checkbox">
+                    <label for="inhuman">Inhuman</label><br>
+
+                    <input name="category" id="mutant" type="checkbox">
+                    <label for="mutant">Mutant</label><br><br>
+                </fieldset>
+
+                <fieldset class="field-margin">
+                    <legend>Gender</legend>
+                    <input name="Gender" id="woman" type="radio">
+                    <label for="woman">Woman</label><br>
+                    <input name="Gender" id="man" type="radio">
+                    <label for="man">Man</label><br>
+                    <input name="Gender" id="nonbinary" type="radio">
+                    <label for="nonbinary">Nonbinary</label><br>
+                    <input name="Gender" id="genderqueer/genderfluid" type="radio">
+                    <label for="genderqueer/genderfluid">Genderqueer/Genderfluid</label><br>
+                    <input name="Gender" id="prefer not to answer" type="radio">
+                    <label for="prefer not to answer">Prefer not to answer</label><br>
+                    <input name="Gender" id="prefer to self describe" type="radio">
+                    <label for="prefer to self describe">Prefer to self describe</label><br><br>
```

Comment:
`br` elements should not be used to fix spacing problems you can control via CSS.
```suggestion
                    <label for="prefer to self describe">Prefer to self describe</label>
```